src/components/ColorSwatch/ColorSwatch.tsx

```diff
@@ -42,14 +58,11 @@ export const ColorSwatch: React.FC<Props> = ({
         className
       )}
       onClick={handleClick}
+      onMouseOver={() => onHover && onHover(color)}
+      onMouseOut={() => onHover && onHover(undefined)}
```

Comment:
This doesn't make a ton of sense to me. "Hover" as a concept implies an on/off callback. I'd just keep these to be onMouseOver/Out

Reply:
welp i merged